packages/editor/src/functions/assetFunctions.ts

```diff
@@ -72,18 +73,37 @@ function findMimeType(file): FileType {
   return fileType
 }
 
-function isValidFileType(file): boolean {
+function isValidFileType(file): { isValid: boolean; errorMessage?: string } {
   const mimeType: FileType = findMimeType(file)
   const fileName = file.name
   const extension = fileName.slice(fileName.lastIndexOf('.')).toLowerCase()
 
   for (const [type, extensions] of Object.entries(supportedFiles)) {
     if (extensions.has(extension)) {
-      return true
+      return {
+        isValid: true
+      }
     }
   }
 
-  throw new Error(unsupportedFileMessage[mimeType])
+  return {
+    isValid: false,
+    errorMessage: unsupportedFileMessage[mimeType]
+  }
+}
+
+function sanitizeFiles(files) {
+  const newFiles: File[] = []
+  for (const file of files) {
+    const newFile = cleanFileNameFile(file)
+    const { isValid, errorMessage } = isValidFileType(newFile)
```

Comment:
is possible, we should check if the file is valid before cleaning the file

Reply:
yea we can do that but I dont think that's necessary here becaues we're not throwing an error and failing the upload. We are just throwing a alert to the user telling them it's not supported.